src/main/java/org/folio/edge/core/Constants.java

```diff
@@ -25,6 +25,23 @@ private Constants() {
   public static final String SYS_API_KEY_SOURCES = "api_key_sources";
   public static final String SYS_RESPONSE_COMPRESSION = "response_compression";
 
+  // System properties for SSL/TLS http server configuration
+  public static final String SYS_HTTP_SERVER_SSL_ENABLED = "http-server.ssl_enabled";
+  public static final String SYS_HTTP_SERVER_KEYSTORE_TYPE = "http-server.keystore_type";
+  public static final String SYS_HTTP_SERVER_KEYSTORE_PROVIDER = "http-server.keystore_provider";
+  public static final String SYS_HTTP_SERVER_KEYSTORE_PATH = "http-server.keystore_path";
+  public static final String SYS_HTTP_SERVER_KEYSTORE_PASSWORD = "http-server.keystore_password";
+  public static final String SYS_HTTP_SERVER_KEY_ALIAS = "http-server.key_alias";
+  public static final String SYS_HTTP_SERVER_KEY_ALIAS_PASSWORD = "http-server.key_alias_password";
+  public static final String SYS_WEB_CLIENT_SSL_ENABLED = "web-client.ssl_enabled";
+  // System properties for SSL/TLS web client configuration
```

Comment:
this comment should be before
public static final String SYS_WEB_CLIENT_SSL_ENABLED = "web-client.ssl_enabled";

Reply:
Probably, we can fix in the scope of other tickets in master branch. If I do changes, there may have merge conflict in next releases and it will not affect in master branch where actual code will be read

Reply:
in this case conflicts can be resolved, mo issues here

Reply:
Ok, fixed